### PR TITLE
straight z 

### PR DESCRIPTION
**Pull request by gourshabrg**
*Describes your contribution*

### 

closes ##262

### CODE REVIEW

File: Fonts/Straight/straight.h [hunk 1]
```diff
@@ -616,18 +616,16 @@ class Straight : public Fonts
...
     */
 
-    /*
-        char **z()
+ char **z()
         {
-            char **character = getCharGrid(ROWS,COLS);
-
-            //Enter the character grid in ROWS X COLS
+            char **character = getCharGrid();
```
Comment (AnshumanMahato) [inline Fonts/Straight/straight.h:621]: Pass the number of rows and columns that you used

File: Fonts/Straight/straight.h [hunk 2]
```diff
@@ -616,18 +616,16 @@ class Straight : public Fonts
...
 
-            return character;
+            character[0][0] ='_';
+            character[1][0] ='/';
+            character[1][1] ='_';
```
Comment (AnshumanMahato) [inline Fonts/Straight/straight.h:625]: Adjust the grid so the z aligns with the rest of the characters. Currently, this is happening.
image

Keeping the top row empty should do the trick.

**Review by AnshumanMahato — CHANGES_REQUESTED** [review]

File: Fonts/Straight/straight.h (hunk 2 above)
Comment (gourshabrg) [reply Fonts/Straight/straight.h:625]: how to solve this issue

**Comment by AnshumanMahato** [comment]
@gourshabrg sorry I missed this one and forgot to reply. Just busy with the festival...

Look what you do... take an extra row for the character and keep the top row empty.. fill ypur character starting from the second row. This should work.

**Pull request closed without merging**